### PR TITLE
[Add education component]Add education component in city file

### PR DESCRIPTION
**Pull request by zhou-1**
Education component includes 3 parts with image shows.

### CODE REVIEW

**Review by gallettilance — CHANGES_REQUESTED** [review]
Why did you change the `city.py` file? You should implement your functions in their respective files.

**Comment by zhou-1** [comment]
> Why did you change the `city.py` file? You should implement your functions in their respective files.

I thought we can "implement" it whatever we want, I will make a change...

**Comment by zhou-1** [comment]
Just deleted the images and changed back to original city file. Also added school and library to city.

**Comment by gallettilance** [comment]
@zhou-1 the images are great actually please feel free to use them. I just ask that you implement that inside their respective files instead of in `city.py`.

**Comment by zhou-1** [comment]
> @zhou-1 the images are great actually please feel free to use them. I just ask that you implement that inside their respective files instead of in `city.py`.

@gallettilance I see, just added images and modified the function a little bit. Hope this will make it better. :)

File: 01-python/education/library.py [hunk 1]
```diff
@@ -1,3 +1,31 @@
+from PIL import Image
+import time
+
 def draw_library():
-    print("library not found")
+img = Image.open("library.png")
```
Comment (gallettilance) [inline 01-python/education/library.py:5]: needs an indent

File: 01-python/education/library.py [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+	im.show()
+	time.sleep(3)
+	im.close()
+	
+    print("+---------------------------------------------------------+")
```
Comment (gallettilance) [inline 01-python/education/library.py:10]: this indent is also off

File: 01-python/education/school.py [hunk 3]
```diff
@@ -1,3 +1,31 @@
...
+import time
+
 def draw_school():
-    print("school not found")
+	img = Image.open("school.jpg")
+	im.show()
+	time.sleep(3)
+	im.close()
+	
+    print("+---------------------------------------------------------+")
```
Comment (gallettilance) [inline 01-python/education/school.py:10]: this indent is off

File: README.md [hunk 4]
```diff
@@ -6,4 +6,6 @@ Code, slides, and notes for Boston University CS506 Fall 2020
 
 ___
 
-The Final Project Repository can be found [here](https://github.com/BU-Spark/CS506-Fall2020-Projects)
\ No newline at end of file
+The Final Project Repository can be found [here](https://github.com/BU-Spark/CS506-Fall2020-Projects)
```
Comment (gallettilance) [inline README.md:9]: please remove these changes

**Review by gallettilance — CHANGES_REQUESTED** [review]

File: 01-python/education/library.py (hunk 1 above)
Comment (zhou-1) [reply 01-python/education/library.py:5]: I don't know why, but they are all looking good in my editor...

File: README.md (hunk 4 above)
Comment (zhou-1) [reply README.md:9]: I will do that.

File: README.md [hunk 5]
```diff
@@ -6,4 +6,4 @@ Code, slides, and notes for Boston University CS506 Fall 2020
 
 ___
 
-The Final Project Repository can be found [here](https://github.com/BU-Spark/CS506-Fall2020-Projects)
\ No newline at end of file
+Finish exercise 3
```
Comment (gallettilance) [inline README.md:9]: It's the opposite that is needed - you should not add the exercise 3 work here.

**Review by gallettilance — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (zhou-1) [reply README.md]: My bad, changed!

File: README.md [hunk 6]
```diff
@@ -3,7 +3,3 @@
 [![Build Status](https://travis-ci.com/gallettilance/CS506-Fall2020.svg?branch=master)](https://travis-ci.com/gallettilance/CS506-Fall2020)
 
 Code, slides, and notes for Boston University CS506 Fall 2020
-
-___
-
-The Final Project Repository can be found [here](https://github.com/BU-Spark/CS506-Fall2020-Projects)
```
Comment (gallettilance) [inline README.md:9]: please add these deleted lines back so that you are not changing the readme when this merges

**Review by gallettilance — CHANGES_REQUESTED** [review]

**Comment by gallettilance** [comment]
also please squash your commits

**Review by zhou-1 — COMMENTED** [review]
> also please squash your commits

@gallettilance Not sure whether I did it correctly, I tried " git rebase -i --root" ("git rebase -i HEAD~N" doesn't work for me)....

File: e -i --root [hunk 7]
```diff
@@ -0,0 +1,47 @@
+[33mcommit 365d21091e7d6d3262f9ff185b49546907a7dd70[m[33m ([m[1;36mHEAD -> [m[1;32mcomponent-branch[m[33m, [m[1;31morigin/component-branch[m[33m)[m
```
Comment (gallettilance) [inline e -i --root:1]: please remove this file

File: README.md [hunk 8]
```diff
@@ -4,6 +4,6 @@
 
 Code, slides, and notes for Boston University CS506 Fall 2020
 
-___
+___
```
Comment (gallettilance) [inline README.md:7]: please remove this change

**Review by gallettilance — CHANGES_REQUESTED** [review]

File: README.md (hunk 8 above)
Comment (zhou-1) [reply README.md:7]: I don't know why this happened, I changed.

**Review by gallettilance — CHANGES_REQUESTED** [review]
Great work! Just need to remove that strange root file and I will approve your PR

**Comment by zhou-1** [comment]
> Great work! Just need to remove that strange root file and I will approve your PR

@gallettilance I deleted it before I git add, commit and push; it is still here? :(

**Comment by zhou-1** [comment]
> Great work! Just need to remove that strange root file and I will approve your PR

@gallettilance I pushed again, hope this time it works!

**Comment by zhou-1** [comment]
Please let me know whether it works. :)

**Pull request closed without merging**